rackcity/api/serializers.py

```diff
@@ -1,8 +1,51 @@
 from rest_framework import serializers
 
-from rackcity.models import Article
+from rackcity.models import (
+    Article,
+    ITInstance,
+    ITModel,
+    Rack,
+    User,
+)
+
 
 class ArticleSerializer(serializers.ModelSerializer):
     class Meta:
         model = Article
-        fields = ('id', 'title', 'content')
\ No newline at end of file
+        fields = ('id', 'title', 'content')
+
+
+class ITInstanceSerializer(serializers.ModelSerializer):
```

Comment:
`ModelSerializer` keeps serializer code more concise, would otherwise be very redundant to field definitions on the model 

see https://www.django-rest-framework.org/tutorial/1-serialization/#using-modelserializers